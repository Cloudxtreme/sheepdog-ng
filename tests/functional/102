#!/bin/bash

# Test vdi deletion of empty volume

. ./common

for i in `seq 0 2`; do
    _start_sheep $i
done

_wait_for_sheep 3

_cluster_format -c 3

$DOG vdi create test 4T
$DOG vdi delete test &

sleep 3
_vdi_list

