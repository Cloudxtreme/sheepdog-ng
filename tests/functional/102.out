QA output created by 102
using backend plain store
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
